packages/block-library/src/rss/editor.scss

```
@@ -5,3 +5,10 @@
 .wp-block-rss__placeholder-form .wp-block-rss__placeholder-input {
 	flex: 1 1 auto;
 }
+// resetting all the various block support styles when skipping block supports in the server-side rendering
+// to prevent the double-up of block support styles within the editor when server-side rendering
```

```suggestion
// Reset all styles when skipping block supports while server-side rendering.
// This prevents issues such as duplicate padding, borders etc.
```

This comment is a little wordy and repetitive. It also needs to follow code style guidelines regarding capitalization and punctuation. 

Perhaps the above suggestion would work better?